pom.xml

```diff
@@ -19,16 +19,18 @@
         <joda-time.version>2.7</joda-time.version>
         <logback.core.version>1.1.2</logback.core.version>
         <dozer.version>5.5.1</dozer.version>
+        <orika.version>1.5.0</orika.version>
```

Comment:
I have to take back my words, you're right. Orika was used in the past, but that has been refactored. I've removed the dependency. Thanks for pointing that out!